src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[5];
+
+        for (int i = 0; i < figures.length; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+        }
+
+        for (Figure figure : figures) {
+            figure.draw();
+        }
```

Comment:
not implemented:
The first half of figures in this array should be generated with random parameters.
The other half of the figures should have the same, default parameters.